http.go

```diff
@@ -17,6 +17,18 @@ import (
 	"github.com/valyala/bytebufferpool"
 )
 
+var (
+	requestBodyMaxLimit  = 0
+	responseBodyMaxLimit = 0
```

Comment:
I would use `-1` as a default so people can set it to `0` to prevent any bodies from being kept.